# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/moz.build
# Commit: c684f79d8c67
# Full Hash: c684f79d8c673f21179fce00358d70845a9692c0
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Rename PWebrtcProxyChannel to PWebrtcTCPSocket. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45098
# ==============================================================================

diff -r 82af7b115f60 -r c684f79d8c67 media/mtransport/ipc/moz.build
--- a/media/mtransport/ipc/moz.build	Tue Sep 17 18:15:23 2019 +0000
+++ b/media/mtransport/ipc/moz.build	Tue Sep 17 18:15:41 2019 +0000
@@ -10,10 +10,10 @@
     'PStunAddrsParams.h',
     'StunAddrsRequestChild.h',
     'StunAddrsRequestParent.h',
-    'WebrtcProxyChannel.h',
-    'WebrtcProxyChannelCallback.h',
-    'WebrtcProxyChannelChild.h',
-    'WebrtcProxyChannelParent.h',
+    'WebrtcTCPSocket.h',
+    'WebrtcTCPSocketCallback.h',
+    'WebrtcTCPSocketChild.h',
+    'WebrtcTCPSocketParent.h',
 ]
 
 UNIFIED_SOURCES += [
@@ -22,16 +22,16 @@
     'ProxyConfigLookupParent.cpp',
     'StunAddrsRequestChild.cpp',
     'StunAddrsRequestParent.cpp',
-    'WebrtcProxyChannel.cpp',
-    'WebrtcProxyChannelChild.cpp',
-    'WebrtcProxyChannelParent.cpp',
-    'WebrtcProxyLog.cpp',
+    'WebrtcTCPSocket.cpp',
+    'WebrtcTCPSocketChild.cpp',
+    'WebrtcTCPSocketLog.cpp',
+    'WebrtcTCPSocketParent.cpp',
 ]
 
 IPDL_SOURCES += [
     'PProxyConfigLookup.ipdl',
     'PStunAddrsRequest.ipdl',
-    'PWebrtcProxyChannel.ipdl',
+    'PWebrtcTCPSocket.ipdl',
 ]
 
 include("/ipc/chromium/chromium-config.mozbuild")